client/src/Components/Layouts/Customers/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Sidebar from '../../CommonComponent/Sidebar';
+export default class Customers extends Component{
+    state={
+        customers:[],
+        name:''
+    }
+    componentDidMount(){
+        axios()
+    }
+    render(){
+        return(
+            <div>
+                <Sidebar/>
+                <Navbar name={this.state.name}/>
+                <Header  />
```

Comment:
the sidebar, navbar, and header put it in the App.jsx, not in the page because it shared between all pages.

Reply:
but we need to do it to create each layout which we have.

Reply:
in app.jsx will create the routes and render the associated  layout just not all conponent